__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+
+describe("GET /api/topics", () => {
+  test("returns an array of topics with slug and description", () => {
+    return request(app)
+      .get("/api/topics")
```

Comment:
Could add error handling to check for invalid endpoints for example /api/katherine